provider/internal/builtin/service_client.go

```diff
@@ -135,89 +135,67 @@ func (p *builtintServiceClient) Evaluate(cap string, conditionInfo []byte) (prov
 		var xmlFiles []string
 		if len(cond.XML.Filepaths) == 0 {
```

Comment:
Where and how, do we handle when there are filepaths that are passed in?

Reply:
yeah, looks like we're missing that branch. GetFiles (https://github.com/konveyor/analyzer-lsp/pull/233/files/f958037f3de788a7c1a90abe15cfed88b9558c05#diff-6335e3249b8d9e8dfe8312aacc749597539e4c377dc0fc779ab68c224fa25f9cR147) is being called with a cond.XML.FilePaths that we already know is empty, we need to move that to an `else` branch for populating `xmlFiles`